.travis.yml

```diff
@@ -2,11 +2,19 @@ language: node_js
 dist: trusty
 sudo: required
 node_js: stable
+env:
+  - SIMPLE_STORAGE_ADDRESS: "0x7ec89063be181f1ad3bcea675bb6413da0f3f50b"
 install:
-  - npm install -g bower
   - npm install
 script:
-  - bower install --production
```

Comment:
This command only installed bower `dependencies`, without `--production` both `dependencies` and `devDependencies` are installed so if at some point we accidentally introduce dependency in `src` to some lib from `devDependencies` we wouldn't find it out.

Reply:
Changed this in the `npm run bower` step in package.json